sql/date-and-time-types.md

```diff
@@ -5,19 +5,56 @@ category: user guide
 
 # Date and time types
 
-The values used to represent date and time types are DATE, TIME, DATETIME, TIMESTAMP, and YEAR. Each of these types has its own range of valid values, and uses a zero value to indicate that it is an invalid value. The TIMESTAMP has an behavior of automatic update, which will be introduced later.
+TiDB supports the following data types to store temporal values: `DATE`, `TIME`, `DATETIME`, `TIMESTAMP`, and `YEAR`. Each of these types has its own range of valid values, and uses a zero value to indicate that it is an invalid value. In addition, the `TIMESTAMP` and `DATETIME` types can automatically generate new time values on modification.
 
-When dealing with date and time value types, please be noted: 
+When dealing with date and time value types, please note: 
+
++ Although TiDB tries to interpret different formats, the date-portion must be in the format of year-month-day (for example, '1998-09-04'), rather than month-day-year or day-month-year.
++ If the year-portion of a date is specified as 2 digits, TiDB converts it based on the following rules:
+  - Values between 70 and 99 is converted to a value between 1970 and 1999
+  - Values between 00 and 69 is converted to a value between 2000 and 2069
++ If a numberic value is needed in the context, TiDB automatically converts the date or time value into a numeric type. For example,
+
+  ```sql
+  mysql> SELECT NOW(), NOW()+0, NOW(3)+0;
+  +---------------------+----------------+--------------------+
+  | NOW()               | NOW()+0        | NOW(3)+0           |
+  +---------------------+----------------+--------------------+
+  | 2012-08-15 09:28:00 | 20120815092800 | 20120815092800.889 |
+  +---------------------+----------------+--------------------+
+  ```
+  
++ TiDB may automatically convert invalid values or values beyond the supported range to a zero value of that type. This behavior is dependent on the SQL Mode set. For example,
+
+  ```sql
+  mysql> set @@sql_mode = '';                               
+  Query OK, 0 rows affected (0.00 sec)
+
+  mysql> show create table t1;
+  +-------+---------------------------------------------------------------------------------------------------------+
+  | Table | Create Table                                                                                            |
+  +-------+---------------------------------------------------------------------------------------------------------+
+  | t1    | CREATE TABLE `t1` (
+    `a` time DEFAULT NULL
+  ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin |
+  +-------+---------------------------------------------------------------------------------------------------------+
+  1 row in set (0.00 sec)
+  
+  mysql> insert into t1 values ('2090-11-32:22:33:44');
```

Comment:
Can we move the show create table above the `set sql_mode`, and then add the same insert with the default sql mode too?  I think that makes for a more interesting example because it contrasts the two behaviors.